.github/workflows/ci.yml

```diff
@@ -482,6 +482,10 @@ jobs:
             !:trino-sqlserver,
             !:trino-test-jdbc-compatibility-old-server,
             !:trino-tests'
+      - name: trino-hdfs isolated JVM tests
+        # Isolate HDFS file system cache concurrency test to avoid flakiness
+        run: |
+          $MAVEN test ${MAVEN_TEST} -pl :trino-hdfs -P test-isolated-jvm-suites
```

Comment:
@hashhar @nineinchnick This and previous step will both write to `trino-hdfs/target/surefire-reports/`. However I do see different `TEST-TestSuite.xml` and `TEST-io.trino.hdfs.TestFileSystemCache.xml` that are relevant for "Upload test report".  Also "Upload test results" is only invoked on failures - so we'll upload the contents for the first profile that failed.  From perspective of CI's further dependencies - is a separate job preferred or this is fine?

Reply:
@nineinchnick can you verify. As i understand we'll have both reports and if first job fails only that report will be uploaded but if the latter fails then both first and second will be uploaded.

I don't think it's a problem though since the worst that can happen is you'll have a report for a successful run as well which doesn't hurt.

Reply:
Yes, I think it's fine as is.

Reply:
Thanks for the input